oak_runtime/src/node/http/mod.rs

```diff
@@ -480,11 +481,27 @@ impl Pipe {
     }
 }
 
+// HTTP requests can either provide JSON formatted labels or protobuf encoded labels. But exactly
+// one of these should be provided. This method checks that exactly one label is provided in a
+// header in the request and extracts it for use for further handling of the request.
 fn get_oak_label(req: &HttpRequest) -> Result<Label, OakStatus> {
```

Comment:
Is this in sync with the logic from the gRPC server pseudo-node? if not, should it be?

Reply:
Yes that's what I mean, also I didn't realise that the gRPC metadata ends up with a different header name, my main point about this logic was that I thought HTTP and gRPC would use the same header name, and therefore some disambiguation is necessary (especially if we plan to eventually merge the two node types together)

Reply:
I think we need to discuss this further.